hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,17 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ $platform == "windows/amd64" ]]; then
+        os::build::version::get_vars
+        go get -u github.com/josephspurrier/goversioninfo
```

Comment:
> Then we install the dep in the environment doing the build first

do you mean following this doc https://github.com/kubernetes/community/blob/master/contributors/devel/godep.md to install dep?